login.css

```diff
@@ -0,0 +1,44 @@
+body{
+  background: url(images/cloudBackground.png) no-repeat center center fixed;
+  -webkit-background-size: cover;
+  -moz-background-size: 100% auto;
+  margin: auto;
+  width: 100%;
+}
+
+.container {
+  padding: 25px;
+  position: absolute;
+  top: 25%;
+  left: 20%;
+
+}
+
+.loginForm{
+  background-color: #EDEDED;
+  padding-top: 10px;
+  padding-bottom: 20px;
+  padding-left: 20px;
+  padding-right: 20px;
+  border-radius: 15px;
+  border-color: #d2d2d2;
+  border-width: 5px;
+  box-shadow: 0 1px 0 #cfcfcf;
+}
+
+.headingLogin {
+  border: 0 solid #fff;
+  border-bottom-width: 1px;
+  padding-bottom: 10px;
+  text-align: center;
+}
+
+.form-control {
+  border-radius: 10px;
+}
+
+.alignment {
+  text-align: center;
+}
+
+p.invalidUser, p.invalidPass{display:none;}
```

Comment:
is this normal style? how about:
`p.invalidUser, p.invalidPass { display: none; } `

Jeffrey to comment

Reply:
Yeah it should have spaces in the brackets.